### PR TITLE
Creating new SDK for ADF new API veriosn

### PR DESCRIPTION
**Pull request by hvermis**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by hvermis** [comment]
Will add the release notes and updates to proj file in a separate commit.

**Comment by dsgouda** [comment]
@hvermis Please link the related REST spec

**Comment by hvermis** [comment]
PR for swagger changes - https://github.com/Azure/azure-rest-api-specs/pull/3270

**Comment by dsgouda** [comment]
@azuresdkci Test this please

**Comment by dsgouda** [comment]
@hvermis Please fix the failing tests

File: src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataFactory</PackageId>
 		<Description>Azure Data Factory V2 is the data integration platform that goes beyond Azure Data Factory V1's orchestration and batch-processing of time-series data, with a general purpose app model supporting modern data warehousing patterns and scenarios, lift-and-shift SSIS, and data-driven SaaS applications. Compose and manage reliable and secure data integration workflows at scale. Use native ADF data connectors and Integration Runtimes to move and transform cloud and on-premises data that can be unstructured, semi-structured, and structured with Hadoop, Azure Data Lake, Spark, SQL Server, Cosmos DB and many other data platforms.</Description>
-		<Version>0.8.0-preview</Version>
+		<Version>1.0.0</Version>
 		<AssemblyName>Microsoft.Azure.Management.DataFactory</AssemblyName>
 		<PackageTags>Microsoft Azure resource management;Data Factory;ADF;</PackageTags>
 		<PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj]: Please update the `PackageReleaseNotes` here

File: src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataFactory</PackageId>
 		<Description>Azure Data Factory V2 is the data integration platform that goes beyond Azure Data Factory V1's orchestration and batch-processing of time-series data, with a general purpose app model supporting modern data warehousing patterns and scenarios, lift-and-shift SSIS, and data-driven SaaS applications. Compose and manage reliable and secure data integration workflows at scale. Use native ADF data connectors and Integration Runtimes to move and transform cloud and on-premises data that can be unstructured, semi-structured, and structured with Hadoop, Azure Data Lake, Spark, SQL Server, Cosmos DB and many other data platforms.</Description>
-		<Version>0.8.0-preview</Version>
+		<Version>1.0.0</Version>
```
Comment (dsgouda) [inline src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj]: Please update the `AssemblyInfo.cs` too

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Review by dsgouda — APPROVED** [review]
LGTM
Please merge the commits into a single commit and this is good to merge

**Pull request merged**